### PR TITLE
Stub for Native Canvas Image

### PR DESCRIPTION
**Pull request by CedricGuillemet**
Native Canvas Image object Stub.
For canvas drawImage method / GUI Image.
Splitting PRs for ease of review.

JS Abstraction PR is here : https://github.com/BabylonJS/Babylon.js/pull/10709

### CODE REVIEW

File: Polyfills/Canvas/Source/Image.cpp [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        Napi::Function func = DefineClass(
+            env,
+            JS_CONSTRUCTOR_NAME,
+            {
+                InstanceAccessor("width", &NativeCanvasImage::GetWidth, nullptr),
+                InstanceAccessor("height", &NativeCanvasImage::GetHeight, nullptr),
+                InstanceAccessor("naturalWidth", &NativeCanvasImage::GetNaturalWidth, nullptr),
+                InstanceAccessor("naturalHeight", &NativeCanvasImage::GetNaturalHeight, nullptr),
+                InstanceAccessor("src", &NativeCanvasImage::GetSrc, &NativeCanvasImage::SetSrc),
+                InstanceAccessor("onload", nullptr, &NativeCanvasImage::SetOnload),
```
Comment (bghgary) [inline Polyfills/Canvas/Source/Image.cpp:28]: Setter and no getter seems weird, but I guess it's unlikely someone will get this.

File: Polyfills/Canvas/Source/Image.cpp [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+namespace Babylon::Polyfills::Internal
+{
+    static constexpr auto JS_CONSTRUCTOR_NAME = "NativeCanvasImage";
+
+    void NativeCanvasImage::CreateInstance(Napi::Env env)
+    {
+        Napi::HandleScope scope{env};
+
+        Napi::Function func = DefineClass(
```
Comment (bghgary) [inline Polyfills/Canvas/Source/Image.cpp:19]: What about `crossOrigin`?

**Review by bghgary — APPROVED** [review]

File: Polyfills/Canvas/Source/Image.cpp (hunk 2 above)
Comment (CedricGuillemet) [reply Polyfills/Canvas/Source/Image.cpp:19]: I don't know yet what to do with `crossOrigin`

Comment (bghgary) [reply Polyfills/Canvas/Source/Image.cpp:19]: That's fine. I noted it since you put everything else except for this.

**Pull request merged**